pkg/oc/lib/graph/buildgraph/analysis/bc.go

```diff
@@ -97,13 +98,17 @@ func FindMissingInputImageStreams(g osgraph.Graph, f osgraph.Namer) []osgraph.Ma
 					// prior to our reaching this point in the code; so there is not need to check for that type vs. ImageStreamTag or ImageStreamImage;
 
 					tagNode, _ := bcInputNode.(*imagegraph.ImageStreamTagNode)
-					imageStream := imageStreamNode.Object().(*imageapi.ImageStream)
-					if _, ok := imageStream.Status.Tags[tagNode.ImageTag()]; !ok {
-
+					imageStream := imageStreamNode.Object().(*imagev1.ImageStream)
```

Comment:
I'm not touching these in this PR, I'm worried what monsters this might uncover. The graph library deserves a big overhaul, but that's a separate topic.